pkgs/by-name/gd/gdmd/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  gdc,
+  perl,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "gdmd";
+  version = "0.1.0-unstable-2024-05-30";
+
+  src = fetchFromGitHub {
+    owner = "D-Programming-GDC";
+    repo = "gdmd";
+    rev = "dc0ad9f739795f3ce5c69825efcd5d1d586bb013";
+    hash = "sha256-Sw8ExEPDvGqGKcM9VKnOI6MGgXW0tAu51A90Wi4qrRE=";
+  };
+
+  patches = [ ./0001-gdc-store-path.diff ];
+
+  buildInputs = [
+    gdc
+    perl
+  ];
+
+  postPatch = ''
+    substituteInPlace dmd-script \
+      --subst-var-by gdc_dir ${gdc}/bin
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 dmd-script $out/bin/gdmd
+    install -Dm644 dmd-script.1 $out/share/man/man1/gdmd.1
```

May or may not be better:

```suggestion
    install -d $out/bin $out/share/man/man1
    make DESTDIR=$out prefix= install
```

Probably the `install -d` should be sent upstream.

---

Thanks, applied a similar change that uses `installFlags`.